doc/nrf/libraries/modem/at_monitor.rst

```diff
@@ -60,6 +65,27 @@ The following code snippet shows how to register a handler that receives ``+CERE
 		printf("Received +CEREG notification: %s", notif);
 	}
 
+The size of the AT monitor library heap can be configured using the :kconfig:`AT_MONITOR_HEAP_SIZE` option.
+
+Direct dispatching
+******************
+
+The AT monitor library supports defining a particular type of monitor which receives the AT notifications in a interrupt service routine.
+Because notifications dispatched to AT monitors in a ISR are not copied onto the AT monitor library heap, this gives a guarantee to the application that there won't be a situation where the library is out of memory to copy the notification.
```

Comment:
```suggestion
Because notifications dispatched to AT monitors in an ISR are not copied onto the AT monitor library heap, this gives a guarantee to the application that there won't be a situation where the library is out of memory to copy the notification.
```